### DISCUSSION TITLE
老大哥，广播不是 bOARdcast, 是 bROAdcast

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#688] [General]
我本来想自己改掉完事的，一搜索发现一堆地方拼错了，我就放弃了

**Comment by Arondight** [comment]
不是吧，这么多年来我一直这么写的……

**Reply by mark9804** [reply]
<img width="296" alt="image" src="https://user-images.githubusercontent.com/9006264/156760197-9b59f6b7-0433-4dda-8dc8-984038bcd7d8.png">

**Reply by Arondight** [reply]
啊这，暂时就这样吧，就当我自创的单词了

**Reply by Arondight** [reply]
改了还得做兼容，要不然我的 fork 会炸，不太想代码里一堆历史包袱，oicq 已经留了一堆 1.x 的包袱了，写错了就错了吧

**Reply by mark9804** [reply]
我本来以为只有几个地方用了广播，后来一看要炸一整条调用链就没动了，但是要好好学习啊老大哥，不然别人写的文档看不懂了

**Reply by Arondight** [reply]
这么多年来我写的和看的居然不是一个单词，而且我还没注意到，也挺离谱的
